app/views/whitehall_migration/document_imports.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :back_link, render_back_link(href: whitehall_migration_path(@whitehall_migration)) %>
+<% title = params[:state] ? "Document Imports: #{params[:state].titleize}" : "Document Imports" %>
+<% content_for :title, title %>
+
+<% if @document_imports.any? %>
+<%= render "document_imports_list", document_imports: @document_imports %>
+<% else %>
+  <p class="govuk-body">
+    There are no document imports
+    <% if params[:state] %>
```

Comment:
Would probably be prudent to check this is an expected state given this is user input.

```suggestion
    <% if WhitehallMigration::DocumentImport.states.keys.include(params[:state]) %>
```

Reply:
This gives the same result regardless of the lookup, so not sure if it is worth the query?